### ISSUE TITLE
Installation fails on dependencies (2)

### ISSUE THREAD

**Issue by datamusee** [AliaksandrSiarohin/first-order-model#21]
Hi,

I cloned the repository and tried to run it, but failed on the dependencies. I'm on Windows 10, installed a Python 3.6.6. I get the following error:
...
Collecting toolz==0.9.0
  Downloading toolz-0.9.0.tar.gz (45 kB)
     |████████████████████████████████| 45 kB 1.1 MB/s
ERROR: Could not find a version that satisfies the requirement torch==1.0.0 (from -r requirements.txt (line 25)) (from versions: 0.1.2, 0.1.2.post1, 0.1.2.post2)
ERROR: No matching distribution found for torch==1.0.0 (from -r requirements.txt (line 25))

**Comment by AliaksandrSiarohin** [comment]
If you don't need to re-train you can install any torch 1.0 and higher.

**Comment by Bonierad** [comment]
Downgrading from python 3.8 to 3.7 helps

**Comment by ucrania** [comment]
I went on this website, they explain how to do it: https://pytorch.org/
In the end I just used this command:

`pip install torch===1.4.0 torchvision===0.5.0 -f https://download.pytorch.org/whl/torch_stable.html                `

**Comment by Jefersonwulf** [comment]
To be able to install version 1.4.0 of the torch I had to install and use the anaconda3 promt as they suggest on the site

**Comment by budidino** [comment]
I ran this to install the specified version on my Win10:
`pip3 install https://download.pytorch.org/whl/cu100/torch-1.0.0-cp37-cp37m-win_amd64.whl`

**Comment by Jefersonwulf** [comment]
> I ran this to install the specified version on my Win10:
> `pip3 install https://download.pytorch.org/whl/cu100/torch-1.0.0-cp37-cp37m-win_amd64.whl`

And did it work? Because I tried anyway to download the torch, but it only came in version 1.0.0, and the code requires version 1.4.0

**Comment by dopesocks** [comment]
guys, what should I do after installing torch 1.4.0 cz when I reran the installation command it still gave the same error. plz help

ERROR: Could not find a version that satisfies the requirement torch==1.0.0 (from -r requirements.txt (line 25)) (from versions: 0.1.2, 0.1.2.post1, 0.1.2.post2)

ERROR: No matching distribution found for torch==1.0.0 (from -r requirements.txt (line 25))

**Comment by AliaksandrSiarohin** [comment]
If you already install torch, you should remove line torch==1.0.0 from the requirements.

**Comment by dopesocks** [comment]
Of course! (I can make an excuse and say I'm new to these stuff, and I am, but I'm just stupid lol)
